tests/utils/couchdb-utils.js

```diff
@@ -59,17 +59,37 @@ export const contacts = () => docs.filter(doc => contactTypes.includes(doc.type)
 
 const getDbByDoc = (id) => Object.keys(docsByDb).filter(dnName => docsByDb[dnName].has(id));
 
-export const editDoc = async (doc) => {
+export const insertDocs = async (documents) => {
+  docs.push(...documents);
+  const db = getDb(dbNames[0]);
+  await db.bulkDocs(documents);
+};
+export const editDoc = async (doc, useOldRev = false) => {
```

Comment:
I think we should not create conflicts by default, so `useOldRev` default value should be `true`.

Reply:
Ok, Thank you for this. I don't know how I miss this. 
For me existentDoc._rev is the actual rev, in this case not creating a conflict. For useOldRev, I mean to use an old rev, not the actual. But my solution is incomplete as I'm not getting an old rev to use in this case. I will try getting the document with `options.revs: Include revision history of the document.`
  ```
const doc = await db.get(docId, { revs: true });
const revisions = doc._revisions;
```
and use:
`revisions.ids[1]`
Is it a good idea to assume there are at least two revisions to the document?
Let me know your thoughts on this, please.

Reply:
You don't actually need an old revision. You can create a conflict by just not using a revision at all. 
so you could just `delete doc._rev` and push it.